packages/twenty-front/src/pages/settings/developers/SettingsDevelopers.tsx

```diff
@@ -19,12 +17,12 @@ const StyledButtonContainer = styled.div`
 
 export const SettingsDevelopers = () => {
   return (
-    <SubMenuTopBarContainer Icon={IconSettings} title="Settings">
+    <SubMenuTopBarContainer
+      Icon={IconCode}
+      title="Developers"
+      actionButton={<SettingsReadDocumentationButton />}
```

Comment:
Not a huge fan of transclusion like this as it is the JSX equivalent of having an any param.